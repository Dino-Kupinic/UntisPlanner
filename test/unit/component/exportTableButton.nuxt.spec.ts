import {it, expect, describe} from "vitest"
import {mountSuspended} from "@nuxt/test-utils/runtime"
import {ExportTableButton} from "#components"

describe("ExportTableButton", () => {
  it("Display button correctly", async () => {
    const component = await mountSuspended(ExportTableButton)
    expect(component.html()).toMatchSnapshot()
  })

  // TODO: Test modal
})
