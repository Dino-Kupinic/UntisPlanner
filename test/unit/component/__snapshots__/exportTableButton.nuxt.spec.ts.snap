// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ExportTableButton > Display button correctly 1`] = `
"<div class="flex justify-around mt-3 mr-3 ml-3 rounded"><button type="button" class="focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 flex-shrink-0 font-medium rounded-md text-sm gap-x-1.5 px-2.5 py-1.5 shadow-sm text-white dark:text-gray-900 bg-primary-500 hover:bg-primary-600 disabled:bg-primary-500 dark:bg-primary-400 dark:hover:bg-primary-500 dark:disabled:bg-primary-400 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-primary-500 dark:focus-visible:outline-primary-400 inline-flex items-center">
    <!--v-if-->Generate
    <!--v-if-->
  </button></div>"
`;
